spec/lib/mpi/service_spec.rb

```diff
@@ -183,6 +183,52 @@
     end
   end
 
+  describe '.find_profile with orch_search' do
+    let(:user) { build(:user, :loa3, user_hash) }
+
+    describe '.find_profile with attributes' do
+      context 'valid request' do
+        let(:user_hash) do
+          {
+            first_name: 'MARK',
+            last_name: 'WEBB',
+            middle_name: '',
+            birth_date: '1950-10-04',
+            ssn: '796104437',
+            edipi: '1013590059'
+          }
+        end
+
+        it 'calls the find profile with an orchestrated search', run_at: 'Thu, 06 Feb 2020 23:59:36 GMT' do
+          allow(SecureRandom).to receive(:uuid).and_return('b4d9a901-8f2f-46c0-802f-3eeb99c51dfb')
+          allow(Socket).to receive(:ip_address_list).and_return([Addrinfo.ip('1.1.1.1')])
+
+          VCR.use_cassette('mpi/find_candidate/orch_search_with_attributes', VCR::MATCH_EVERYTHING) do
+            Settings.mvi.vba_orchestration = true
```

Comment:
Could you add a mock for this instead of actually changing the Settings values: `allow(Settings.mvi).to receive(:vba_orchestration).and_return(true)` ? 

I'm trying to remove singleton editing from specs in general, since it leads to unexpected behavior (for example, in this case it appears that you cleaned up properly, however, if we had actually expected `vba_orchestration` to be set to `true` for everything, then setting it to `false` at the end here would unexpectedly change downstream specs)